tests/test_oml/test_datasets/test_audio.py

```diff
@@ -0,0 +1,49 @@
+import random
```

Comment:
pls, use `@pytest.mark.needs_optional_dependency` if you need torchaudio for some tests below

Reply:
done